lib/maven-utils.js

```diff
@@ -21,6 +21,8 @@ class MavenUtils {
 	}
 }
 
+MavenUtils.prototype.$ = require('jquery');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier name '$' is too short. (< 3)](https://www.codacy.com/app/connor-bray/atom-maven/file/2738198025/issues/source?bid=3352650&fileBranchId=3353288#l24)